curation/views.py

```diff
@@ -138,12 +141,62 @@ def _str_to_int(value):
         gene=gene,
     )
     if genomic_coords:
-        response['gene'] = genomic_coords.get("gene", None)
-        response['chr'] = genomic_coords.get("chr", None)
+        gene = genomic_coords.get('gene', None)
+        response['gene'] = gene
+        response['gene_id'] = normalize_gene(gene)['concept_id']
+        chr = genomic_coords.get("chr", None)
+        response['chr'] = chr
+        sequence_id = get_sequence_id(f'refseq:{chr}')
+        response['sequence_id'] = sequence_id['sequence_id']
         response['start'] = genomic_coords.get("start", None)
         response['end'] = genomic_coords.get("end", None)
         response['start_exon'] = genomic_coords.get("start_exon", None)
         response['end_exon'] = genomic_coords.get("end_exon", None)
         return response
     else:
         return {}
+
+
+@app.route('/sequence/<in_sequence>')
+def get_sequence_id(in_sequence: str) -> Dict:
```

Comment:
I'd suggested using a better name. 
```suggestion
def get_sequence_id(input_sequence: str) -> Dict:
```